### PR TITLE
SPCTR-2283 info box cta button hover fix

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description
[Info Box] When adding CTA, the button hover color is getting changed if the hovering outside of the button borders

### Screenshots
Before changes: https://d.pr/v/8xAdZE
After changes: https://d.pr/v/yVk0pg

### Types of changes
Bug-fix: made changes in style.scss to fix button hover issue

### How has this been tested?
Before changes: https://d.pr/v/8xAdZE
After changes: https://d.pr/v/yVk0pg

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: src/blocks/info-box/components/CallToAction.js [hunk 1]
```diff
@@ -48,7 +49,7 @@ const InfoBoxCta = ( props ) => {
 							ctaIconOutput }
 					</a>
 				) }
-
+
```
Comment (bsf-zanev) [inline src/blocks/info-box/components/CallToAction.js:52]: These extra tab spaces aren't needed.

**Review by bsf-zanev — CHANGES_REQUESTED** [review]
Please address the comments below.

File: src/blocks/info-box/components/CallToAction.js (hunk 1 above)
Comment (rajmohanreddy774) [reply src/blocks/info-box/components/CallToAction.js]: changes done

**Review by bsf-zanev — APPROVED** [review]
Looks good to me.

File: src/blocks/info-box/style.scss [hunk 2]
```diff
@@ -500,3 +500,7 @@ a.uagb-infbox__link-to-all {
 .uagb-disable-link {
 	pointer-events: none;
 }
+
+.wp-block-button {
```
Comment (RajkiranBagal) [inline src/blocks/info-box/style.scss:504]: wp-block-button is a very general class that is used in multiple other blocks, this CSS may apply to other unwanted blocks & cause issues.

If you need CSS only for info box then use info box-specific classes or add CSS to existing selector in the file.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

**Review by RajkiranBagal — APPROVED** [review]

File: src/blocks/info-box/components/CTA.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ const InfoBoxCta = ( props ) => {
 
 	let ctaIconOutput = '';
 	if ( attributes.showCtaIcon && attributes.ctaIcon !== '' ) {
-		ctaIconOutput = renderSVG( attributes.ctaIcon, setAttributes );
```
Comment (RajkiranBagal) [inline src/blocks/info-box/components/CTA.js:19]: Why is this removed?

File: src/blocks/info-box/components/CTA.js [hunk 4]
```diff
@@ -41,7 +41,6 @@ const InfoBoxCta = ( props ) => {
 								onChange={ ( value ) => {
 									setAttributes( { ctaText: value } );
 								} }
-								allowedFormats={ [] } // Removed the WP default link/bold/italic from the toolbar for button.
```
Comment (RajkiranBagal) [inline src/blocks/info-box/components/CTA.js:44]: Why is this removed?

File: src/blocks/info-box/components/CTA.js [hunk 5]
```diff
@@ -73,7 +72,6 @@ const InfoBoxCta = ( props ) => {
...
```
Comment (RajkiranBagal) [inline src/blocks/info-box/components/CTA.js:76]: Why is this removed?

File: src/blocks/info-box/components/CTA.js (hunk 3 above)
Comment (rajmohanreddy774) [reply src/blocks/info-box/components/CTA.js]: added back

File: src/blocks/info-box/components/CTA.js (hunk 5 above)
Comment (rajmohanreddy774) [reply src/blocks/info-box/components/CTA.js]: added

**Review by supriyajjadhav — APPROVED** [review]

**Review by RajkiranBagal — APPROVED** [review]

**Pull request merged**